templates/java/org/prism/Nodes.java.erb

```diff
@@ -135,27 +135,27 @@ public abstract class Nodes {
         protected abstract String toString(String indent);
     }
 <%# FLAGS -%>
-    <%- flags.each do |group| -%>
+    <%- flags.each do |flag| -%>
 
     /**
-     * <%= Prism::Template::JavaDoc.escape(group.comment) %>
+     * <%= Prism::Template::JavaDoc.escape(flag.comment) %>
      */
-    public static final class <%= group.name %> implements Comparable<<%= group.name %>> {
-        <%- group.values.each_with_index do |value, index| -%>
+    public static final class <%= flag.name %> implements Comparable<<%= flag.name %>> {
+        <%- flag.values.each_with_index do |value, index| -%>
 
         // <%= value.comment %>
-        public static final short <%= value.name %> = 1 << <%= index %>;
+        public static final short <%= value.name %> = 1 << <%= index + 2 %>;
```

Comment:
Ah yes this makes sense.
I did https://github.com/ruby/prism/pull/2956/commits/cf6334e7992ede75fc15ca663c8d0bca379cd4a2 so common flags are not serialized for all nodes, but kept the shift for consistency&simplicity, so we still need to `+ 2` when checking them.